### ISSUE TITLE
Accessibility check with screen reader - Goals and Conversations tab

### ISSUE THREAD

**Issue by phoebesolomon** [bcgov/performance#270]

**Comment by phoebesolomon** [comment]
Notes from accessibility check:

**Log In Page:**

- “Sign in to start your session” should be read aloud
- Email and password field should be announced

**Welcome Page:**

- Cannot use the tab key to navigate to the page content
- Welcome should be announced

**Goals page:**

- ‘Create new goal’ is announced as a button, but add goal from library is announced as a link
- Delete icon on goals card needs a description
- ‘Goal created by’ message on goal card is not announcable 

**Individual Goal page:**

- Cannot tab through content of the goal, it will only recognize the status dropdown menu and comments section

**Create New Goal Modal:**

- Hover text for what, why, how, etc is not being read out loud

**Goal Library Page:**

- ‘Edit text’ should be changed to ‘search bar’
- Page title and description are not being announced by the screen reader
- Column titles in ‘your supervisor’ and ‘your organization’ dropdowns are not being announced, and you cannot hover over them using the tab key
- Column content for ‘date added’ and ‘created by’ is not being announced

**Goals Page:**

- Needs a title
- Url should include ‘my goals’

**My Supervisor’s goals:**

- Page description is not being read
- Goal created by is not being read
- Comments field on individual goal page needs a description - title not being announced
- Cannot tab through any of the content in the individual goal page

**My Goals Page:**

- Titles not announced - cannot tab through

**Upcoming Conversations Page:**

- Warning message at the top is not read aloud, cannot hover over it with tab key
- Content on conversation cards is not being read aloud, only the view and delete buttons

**Conversation template modal:**

- Goal setting template title should be in blue banner for consistency with other modals
- Titles for all sections of the template are not being read out loud - only buttons
